01-import_and_maxfilter.py

```diff
@@ -176,31 +177,44 @@ def find_bad_channels(raw, subject, session, task, run):
     tsv_data.to_csv(bads_tsv_fname, sep='\t', index=False)
 
 
-def apply_maxwell_filter(raw, subject, session, dev_head_t):
-    msg = 'Applying Maxwell filter.'
-    logger.info(gen_log_message(message=msg, step=1,
-                                subject=subject, session=session))
+def load_data(bids_basename):
+    # read_raw_bids automatically
+    # - populates bad channels using the BIDS channels.tsv
+    # - sets channels types according to BIDS channels.tsv `type` column
+    # - sets raw.annotations using the BIDS events.tsv
 
-    # Warn if no bad channels are set before Maxfilter
-    if not raw.info['bads']:
-        msg = '\nFound no bad channels. \n '
-        logger.warn(gen_log_message(message=msg, subject=subject,
-                                    step=1, session=session))
+    params = _parse_bids_filename(bids_basename, verbose=False)
+    subject = params['sub']
+    session = params['ses']
 
-    if config.mf_st_duration:
-        msg = '    st_duration=%d' % (config.mf_st_duration)
-        logger.info(gen_log_message(message=msg, step=1,
-                                    subject=subject, session=session))
+    extra_params = dict()
+    if config.allow_maxshield:
+        extra_params['allow_maxshield'] = config.allow_maxshield
 
-    raw_sss = mne.preprocessing.maxwell_filter(
-        raw,
-        calibration=config.mf_cal_fname,
-        cross_talk=config.mf_ctc_fname,
-        st_duration=config.mf_st_duration,
-        origin=config.mf_head_origin,
-        destination=dev_head_t)
+    raw = read_raw_bids(bids_basename=bids_basename,
+                        bids_root=config.bids_root,
+                        extra_params=extra_params,
+                        kind=config.get_kind())
+
+    # XXX hack to deal with dates that fif files cannot handle
```

Comment:
I don't get this comment. What's the issue?

Reply:
I'm not entirely sure, this comment was not inserted by me 😅